draft-ietf-mls-protocol.md

```diff
@@ -4262,8 +4258,11 @@ occurs:
   received, or the most recent Update if there are no Removes.
 
 * It contains multiple Add proposals that contain KeyPackages that represent the same
-  client according to the application (for example, identical KeyPackages or KeyPackages
-  sharing the same Credential).
+  client according to the application (for example, identical signature keys).
+
+* It contains an Add proposal with a KeyPackage that represents a client already
+  in the group according to the application, which isn't being removed by
+  another proposal in the same Commit.
```

Comment:
```suggestion
  in the group according to the application, unless there is a Remove proposal
  in the list removing the matching client from the group.
```

Clarifies what it means for a client to be "removed by another proposal", and keeps it scoped to the proposal list.